docs/modules/ROOT/pages/index.adoc

```diff
@@ -610,28 +617,41 @@ The `signature` field of an *<<_identity_assertion,identity assertion>>* MUST co
-. Compute a hash of the serialized _<<C2PA claim>>_ data structure using the hash algorithm described by `expected_claim.alg`.
+. Compute a hash of the serialized _<<C2PA claim>>_ data structure using the hash algorithm described by `expected_partial_claim.alg`.
 
-The resulting hash MUST exactly match the value present in `expected_claim.hash`. The `cawg.identity.expected_claim.mismatch` error code SHALL be used to report assertions that contain other values.
+The resulting hash MUST exactly match the value present in `expected_partial_claim.hash`. The `cawg.identity.expected_partial_claim.mismatch` error code SHALL be used to report assertions that contain other values.
 
-==== Validating the `expected_claim_generator` value
+==== Validating the `expected_claim_generator` field
 
-If the *<<_identity_assertion,identity assertion>>* contains an `expected_claim_generator` value in `signer_payload`, it MUST be validated as follows:
+If the *<<_identity_assertion,identity assertion>>* contains an `expected_claim_generator` field in `signer_payload`, it MUST be validated as follows:
 
 . Locate the end-entity signing certificate that is present in the _<<C2PA claim>>_ signature. It can be located as described in link:https://c2pa.org/specifications/specifications/2.0/specs/C2PA_Specification.html#x509_certificates[Section 14.6, “X.509 certificates,”] of the C2PA technical specification.
 . Identify the CBOR bytes specific to this signing certifcate. If necessary, any re-serialization should be performed using link:++https://www.rfc-editor.org/rfc/rfc8949.html#name-core-deterministic-encoding++[Section 4.2.1, “Core deterministic encoding,”] of RFC 8949: Concise binary object representation.
 . Compute a hash of the serialized end-entity certificate using the hash algorithm described by `expected_claim_generator.alg`.
 
 The resulting hash MUST exactly match the value present in `expected_claim_generator.hash`. The `cawg.identity.expected_claim_generator.mismatch` error code SHALL be used to report assertions that contain other values.
 
+==== Validating the `expected_countersigners` field
+
+If the *<<_identity_assertion,identity assertion>>* contains an `expected_countersigners` field in `signer_payload`, it MUST be validated as follows:
+
+. For every _other_ *<<_identity_assertion,identity assertion>>* that is present in the <<C2PA Manifest>>:
+.. Find the `signer_payload` map in that *<<_identity_assertion,identity assertion>>*. Remove any `expected_countersigners` field that is found in that `signer_payload` map. This will be referred to in the rest of this process as the _partial signer payload._
+.. Search the `expected_countersigners` list for an entry where `partial_signer_payload` matches the just-computed _partial signer payload._ If no such entry can be found, stop validating and return the `cawg.identity.unexpected_countersigner` error code.
```

Comment:
I would just disambiguate this further to make it clear that  'search the 'expected_countersigners' list *in the current identity assertion being validated* for an entry where 'partial_signer_payload' matches the just-computed _partial signer payload_

I say this because the control loop these steps are wrapped in makes 'every_other identity assertion' the natural subject and this is an exception to that.